tests/__fixtures__/cardanoSignTransaction.js

```diff
@@ -18,6 +18,11 @@ const SAMPLE_INPUTS = {
         prev_hash: '3b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b7',
         prev_index: 0,
     },
+    shelley_input_2: {
+        path: "m/1852'/1815'/0'/0/1",
+        prev_hash: '3b40265111d8bb3c3c608d95b3a0bf83461ace32d79336579a1939b3aad1c0b7',
```

Comment:
Maybe `prev_hash` should be changed too.

Reply:
Changed.